docs/webui.md

```diff
@@ -160,11 +160,12 @@ Click on the `CREATE CATALOG` button displays the dialog to create a catalog.
 Creating a catalog requires these fields:
 
 1. **Catalog name**(**_required_**): the name of the catalog
-2. **Type**(**_required_**): `relational`/`fileset`/`messaging`, the default value is `relational`
+2. **Type**(**_required_**): `relational`/`fileset`/`messaging`/`model`, the default value is `relational`
 3. **Provider**(**_required_**):
     1. Type `relational` - `hive`/`iceberg`/`mysql`/`postgresql`/`doris`/`paimon`/`hudi`/`oceanbase`
     2. Type `fileset` - `hadoop`
     3. Type `messaging` - `kafka`
+    4. Type `model` have no provider
```

Comment:
"...has no provider"

Reply:
If it has no provider, then provider is also not required?

Reply:
Required for type `relational /fileset/messaging`.